webapp-django/crashstats/crashstats/management/commands/verifyprocessed.py

```diff
@@ -92,16 +96,25 @@ def check_crashids(entropy_chunk, date):
         page_iterator = paginator.paginate(Bucket=bucket, Prefix=raw_crash_key_prefix)
 
         for page in page_iterator:
-            for item in page.get("Contents", []):
-                raw_crash_key = item["Key"]
-                crash_id = raw_crash_key.split("/")[-1]
+            # NOTE(willkg): Keys look like /v2/raw_crash/ENTRPOY/DATE/CRASHID
+            crash_ids = [
+                item["Key"].split("/")[-1] for item in page.get("Contents", [])
+            ]
+
+            if not crash_ids:
```

Comment:
I don't understand how it's possible to get a page that has no items in it. I would think the page_iterator would be smarter than to have us request an empty page. It's weird.